src/core/CCList.mli

```diff
@@ -24,74 +24,75 @@ val is_empty : _ t -> bool
     @since 0.11 *)
 
 val map : ('a -> 'b) -> 'a t -> 'b t
-(** Safe version of {!List.map}. *)
+(** [map f [a0; a1; ⋯ an]] applies function [f] in turn to [a0; a1; ⋯ an].
+    Safe version of {!List.map}. *)
 
 val (>|=) : 'a t -> ('a -> 'b) -> 'b t
-(** Infix version of [map] with reversed arguments.
+(** [l >|= f] is the infix version of [map] with reversed arguments.
     @since 0.5 *)
 
 val append : 'a t -> 'a t -> 'a t
-(** Safe version of {!List.append}.
-    Concatenate two lists. *)
+(** [append l1 l2] returns the list that is the concatenation of [l1] and [l2].
+    Safe version of {!List.append}. *)
 
 val cons_maybe : 'a option -> 'a t -> 'a t
 (** [cons_maybe (Some x) l] is [x :: l].
     [cons_maybe None l] is [l].
     @since 0.13 *)
 
 val (@) : 'a t -> 'a t -> 'a t
-(** Like [append].
-    Concatenate two lists. *)
+(** [l1 @ l2] is like [append].
+    Concatenate the two lists [l1] and [l2]. *)
 
 val filter : ('a -> bool) -> 'a t -> 'a t
-(** Safe version of {!List.filter}.
-    [filter p l] returns all the elements of the list [l]
+(** [filter p l] returns all the elements of the list [l]
     that satisfy the predicate [p].  The order of the elements
-    in the input list is preserved. *)
-
+    in the input list [l] is preserved.
+    Safe version of {!List.filter}. *)
+    
 val fold_right : ('a -> 'b -> 'b) -> 'a t -> 'b -> 'b
-(** Safe version of [fold_right].
-    [fold_right f [a1; ...; an] b] is
-    [f a1 (f a2 (... (f an b) ...))].  *)
+(** [fold_right f [a1; ⋯ an] b] is
+    [f a1 (f a2 ( ⋯ (f an b) ⋯ ))].
+    Safe version of {!List.fold_right}. *)
 
 val fold_while : ('a -> 'b -> 'a * [`Stop | `Continue]) -> 'a -> 'b t -> 'a
-(** Fold until a stop condition via [('a, `Stop)] is
+(** [fold_while f acc l] folds until a stop condition via [('a, `Stop)] is
     indicated by the accumulator.
     @since 0.8 *)
 
 val fold_map : ('acc -> 'a -> 'acc * 'b) -> 'acc -> 'a list -> 'acc * 'b list
-(** [fold_map f init l] is a [fold_left]-like function, but it also maps the
+(** [fold_map f acc l] is a [fold_left]-like function, but it also maps the
```

Comment:
Why did you rename `init` to `acc`? I, personally, see this value as the initial value of the fold.